### PR TITLE
support config minimal wait duration

### PR DESCRIPTION
**Pull request by glorv**

### CODE REVIEW

**Review by SpadeA-Tang — APPROVED** [review]

File: src/limiter.rs [hunk 1]
```diff
@@ -163,6 +169,12 @@ impl<C: Clock> Builder<C> {
         self
     }
 
+    /// Set the minimum wait duration.
```
Comment (kennytm) [inline src/limiter.rs]:
```suggestion
    /// Sets the minimum wait duration when the speed limit was exceeded.
    ///
    /// The default value is same as the refill period.
```

**Review by kennytm — COMMENTED** [review]
consider adding some tests into `mod tests_with_manual_clock` to document the behavior of the new config.

**Comment by glorv** [comment]
> consider adding some tests into `mod tests_with_manual_clock` to document the behavior of the new config.

Done.

**Comment by glorv** [comment]
friendly ping @kennytm  PTAL

**Review by nolouch — APPROVED** [review]

**Review by kennytm — APPROVED** [review]

**Pull request merged**